binding/android/build.gradle

```diff
@@ -0,0 +1,26 @@
+ext {
+    defaultTargetSdkVersion = 31
+}
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:4.2.2"
+        classpath "com.github.dcendents:android-maven-gradle-plugin:2.1"
```

Comment:
this is not required now: take a look at this PR https://github.com/Picovoice/cobra/pull/133